src/main/resources/google_checks.xml

```diff
@@ -16,7 +16,10 @@
     Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
  -->
 
-<module name = "Checker">
+<module name="Checker">
```

Comment:
Please move formatting to separate commit "minor: formatting" keep it in same PR.

Reply:
moved to a separate commit as per your suggestion